backend/data/src/main/kotlin/io/tolgee/configuration/tolgee/AuthenticationProperties.kt

```diff
@@ -119,6 +119,7 @@ class AuthenticationProperties(
         "When enabled, all images are served with an access token valid for\n" +
         "a set period of time to prevent unauthorized access to images.",
   )
+  var createDemoForInitialUser: Boolean = true,
```

Comment:
This now has DocProperty of `securedImageRetrieval`, please fix it and add correct one